packages/app-store/routing-forms/lib/FieldTypes.ts

```diff
@@ -0,0 +1,30 @@
+export const FieldTypes = [
```

Comment:
Was needed because it was used by getQueryBuilderConfig and the tests were failing for it due to it being imported from a page component.